src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Listeners/QueueListener.cs

```diff
@@ -141,17 +143,29 @@ public Task StartAsync(CancellationToken cancellationToken)
         {
             ThrowIfDisposed();
             _timer.Start();
+            _logger.LogDebug($"Storage queue listener started: {_details.Value}");
```

Comment:
We use internal queue for blob triggers - after detecting a new blob we create a message in internal queue and then execute the function based on internal queue. Do we use the same "QueueListener" for the purpose? Will we log the details for internal queue?

Reply:
It looks like yes, we do :/ I could add a flag to the constructor of `QueueListener`, e.g. `shouldLogDetails`, which is `false` for this scenario and `true` for all others. Let me know what you think

Reply:
I think it's ok, it can be also be useful info for us when we starting/stopping listening the internal queue

Reply:
Sounds good!